postgres/init.sls

```diff
@@ -48,9 +48,11 @@ postgresql-running:
       - cmd: postgresql-cluster-prepared
 
 {% if postgres.pkgs_extra %}
-postgresql-extra-pkgs-installed:
+{% for pkg in postgres.pkgs_extra %}
```

Comment:
Unsure about this one. According to our existing pillar data as well as that the defaults.yml does not actually provide an example, it is to assume that 'pkgs_extra' is expected to be a list. Therefore the loop to install each package individually is a safe way to handle this.